lib/matplotlib/tests/test_constrainedlayout.py

```diff
@@ -387,3 +387,16 @@ def test_colorbar_location():
     fig.colorbar(pcm, ax=axs[-2, 3:], shrink=0.5, location='top')
     fig.colorbar(pcm, ax=axs[0, 0], shrink=0.5, location='left')
     fig.colorbar(pcm, ax=axs[1:3, 2], shrink=0.5, location='right')
+
+
+def test_hidden_axes():
+    # test that if we make an axes not vidsdible that constrained_layout
+    # still works.  Note the axes still takes space in the layout
+    # (as does a gridspect slot that is empty)
```

Comment:
```suggestion
    # (as does a gridspec slot that is empty)
```

Reply:
I have one of the accursed mac keyboards... ;-)